lessonbuilder/tool/src/webapp/templates/ShowPage.html

```diff
@@ -65,12 +65,9 @@
  <script type="text/javascript" src="$context/js/jquery.cluetip.min.js"></script>
  <link rel="stylesheet" href="$context/css/jquery.cluetip.css" type="text/css" />
  <link rel="stylesheet" href="/library/editor/ckextraplugins/folder-listing/css/file-tree.css" type="text/css"/>
- <link href="/library/webjars/fullcalendar/4.3.1/core/main.min.css" rel="stylesheet" />
- <link href="/library/webjars/fullcalendar/4.3.1/daygrid/main.min.css" rel="stylesheet" />
- <link href="/library/webjars/fullcalendar/4.3.1/timegrid/main.min.css" rel="stylesheet" />
- <script src="/library/webjars/fullcalendar/4.3.1/core/main.min.js"></script>
- <script src="/library/webjars/fullcalendar/4.3.1/daygrid/main.min.js"></script>
- <script src="/library/webjars/fullcalendar/4.3.1/timegrid/main.min.js"></script>
+ <link href="/library/webjars/fullcalendar/5.8.0/main.min.css" rel="stylesheet" />
+ <script src="/library/webjars/fullcalendar/5.8.0/main.min.js"></script>
+ <script src="/library/webjars/fullcalendar/5.8.0/locales-all.min.js"></script>
```

Comment:
Should we have the git sha on the end of these urls? Cache busting?

Reply:
Could you elaborate?

Reply:
actually, I don't think it matters. You're using a versioned js anyway. Ignore me :)